sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/QueryClient.cs

```diff
@@ -41,8 +40,8 @@ internal QueryClient(QueryRestClient queryRestClient, ClientDiagnostics clientDi
         /// <param name="endTime">End timestamp of the time range. Events that match this timestamp are excluded.</param>
         /// <param name="options">Optional parameters to use when querying for events.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>The pageable list <see cref="AsyncPageable{QueryResultPage}"/> of query result frames.</returns>
-        public virtual AsyncPageable<QueryResultPage> GetEventsAsync(
+        /// <returns>The <see cref="QueryAnalyzer"/> object that can be used to retrieve the pageable list <see cref="AsyncPageable{TimeSeriesPoint}"/>.</returns>
+        public virtual QueryAnalyzer CreateEventsQueryAnalyzer(
```

Comment:
why is called Create?

Reply:
Because it creates a new instance of QueryAnalyzer. The QueryAnalyzer is then used to do the actual querying (by calling the rest endpoints)

Reply:
I would add some more context in the summary then to say something like .. retrieve raw events using QueryAnalyzer created by ...